pcdsdevices/lakeshore.py

```diff
@@ -0,0 +1,87 @@
+"""
+Classes for lakeshore temperature controller
+"""
+import logging
+
+from ophyd import Component as Cpt
+from ophyd import Device, EpicsSignal, EpicsSignalRO
+from ophyd import FormattedComponent as FCpt
+
+from .interface import BaseInterface
+
+logger = logging.getLogger(__name__)
+
+
+class Heater(BaseInterface, Device):
+    """
+    HeaterState Channel Object.
+
+    Parameters
+    ----------
+    channel_prefix : str
+        The EPICS base of the HeaterState Channel. E.g.: ``
+    """
+    htr_state = FCpt(EpicsSignal, '{prefix}:GET_RANGE_{channel}', write_pv='{prefix}:PUT_RANGE_{channel}', kind='normal', doc='heater range')
+    htr_state_rbv = FCpt(EpicsSignalRO, '{prefix}:GET_HTRSTAT_{channel}', kind='normal')
+
+    # insert tab white list
+
+    def __init__(self, prefix, channel, **kwargs):
+        self.channel = channel
+        super().__init__(prefix, **kwargs)
+
+
+class TemperatureSensor(BaseInterface, Device):
+    input_name = FCpt(EpicsSignalRO, '{prefix}:GET_INNAME_{channel}', kind='config')
+    temp = FCpt(EpicsSignalRO, '{prefix}:GET_TEMP_{channel}', kind='normal')
+    units = FCpt(EpicsSignal, '{prefix}:GET_UNITS_{channel}', write_pv=':PUT_UNITS_{channel}', kind='normal')
+    sensor_type = FCpt(EpicsSignalRO, '{prefix}:GET_SENSOR_{channel}', kind='normal')
+
+    def __init__(self, prefix, channel, **kwargs):
+        self.channel = channel
+        super().__init__(prefix, **kwargs)
+
+
+class Lakeshore336(BaseInterface, Device):
+    """
+    two loops
+    4 temperature sensors
+    also include device control mode
+    """
```

Comment:
I think it'd appropriate/helpful for this main docstring to be fleshed out more

```
"""
This class supports the Lakeshore Model 336 Temperature controller.

It includes:

- 2 temperature loops
- 4 temperature sensors
"""
```
etc.